examples/common_patterns_for_connectors/priority_first_sync_for_high_volume_initial_syncs/connector.py

```diff
@@ -0,0 +1,192 @@
+# Priority-first sync, pfs for short, is very helpful for high volume historical syncs. It is a sync strategy
+# which prioritises fetching the most recent data first so that it's quickly ready for you to use.
+# This is a simple example of how the pfs strategy looks like.
```

Comment:
for the above 3 lines match those in the readme - once they have ben finalized.

Reply:
Done.